CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
 cmake_minimum_required(VERSION 3.4)
-project(libuv LANGUAGES C)
 
 cmake_policy(SET CMP0057 NEW) # Enable IN_LIST operator
 cmake_policy(SET CMP0064 NEW) # Support if (TEST) operator
+cmake_policy(SET CMP0091 NEW) # Enable MSVC_RUNTIME_LIBRARY setting to allow statically linking the C runtime
```

Comment:
Not available until 3.15, which exceeds the `cmake_minimum_required` value

Reply:
added `if (POLICY)` command for checking